client/src/styles/bootstrap_ext/_button.scss

```
@@ -39,6 +39,10 @@ $borderRadius: 1000px;
   margin-right: 12px;
 }
 
+.btn-with-menu-options .btn-with-menu-margin {
+  margin-right: 12px;
+}
+
```

I'm guessing that the idea is that the margin should be the same for both `svg` and elements with the specified class. I think in that case, it would be better to combine the definitions:

```suggestion
.btn-with-menu-options {
    svg, .btn-with-menu-margin {
      margin-right: 12px;
     }
}
```
(Not 100% sure on that syntax, but should be something similar.)

---

Ah yes, good point. I'll harmonize that CSS rules with the previous one on `svg` elements